frontend/packages/shared/src/version-control/CloneModal.tsx

```diff
@@ -88,7 +85,11 @@ export function CloneModal(props: ICloneModalProps) {
         )}
         <>
           <div className={classes.blackText}>{t('sync_header.clone_https')}</div>
-          <TextField id='repository-url-form' value={repositoryGitPath(org, app)} readOnly />
+          <TextField
+            id='repository-url-form'
+            value={gitUrl}
+            readOnly
+          />
         </>
         {canCopy() && (
```

Comment:
Nice, looks like you refactored canCopy to be a variable instead of a method. Now we don't need to call canCopy here. 👏 
```suggestion
        {canCopy && (
```